.circleci/config.yml

```diff
@@ -321,15 +333,18 @@ jobs:
           #   src/test/java/org/pmiops/*********/monitoring/LogsBasedMetricsServiceTest.java src/test/java/org/pmiops/*********/monitoring/MonitoringServiceTest.java
           # Example of Gradle command after formatting:
           #   gradle test --tests org.pmiops.*********.monitoring.LogsBasedMetricsServiceTest --tests org.pmiops.*********.monitoring.MonitoringServiceTest
+          #
+          # We exclude the :genomics:test task below because otherwise gradle will attempt to run all the regular API unit tests under the genomics
+          # submodule and will be unable to find them and will barf
```

Comment:
I don't understand - why would the below glob pickup the genomics test files? Or is that not the issue?

Reply:
That is not the issue. 

`gradle test`, by default, runs all subtasks of `gradle test`. This, generally, makes sense - it's common for repos to have, like, a `unitTest` and `integrationTest` subtask and you'd want `gradle test` to run them all.  In this case, `gradle test` runs `:test`, `:appengine-cron-emulator:test`, and `:genomics:test`, since the latter two are submodules of the top-level gradle project.

The issue is that running this command like this passes `$GRADLE_ARGS` to every one of those subtasks with no regard for whether those tests exist in those projects. You end up getting `gradle test :gradle:test --tests org.pmiops.workbench.DoesNotExistInThisClasspathTest`. 

What I'm doing here isn't excluding the genomics test modules from being handed to `gradle test`, I'm excluding the genomics test subtask from executing. This command won't grab the genomics tests anyways because of how `circleci tests glob` works - it's not a regex on paths, it just starts looking from `working_directory`, and since the genomics tests are not under `api/src/test/java` they won't get picked up.

Hope that all makes sense. I've created another CircleCI step above to run just the genomics tests.

Reply:
I see, thanks. My preferred fix would be `gradle test` --> `gradle :test` on this line (no -x), this only picks up the root test task which would have been more accurate to begin with but didn't matter until now. This way you're not needing to list out every subproject that has tests. I tested locally that this works.

I would also clarify the above comment that it pertains specifically to the test filtering. If you filter down to 0 tests for a particular gradle test target, that's an error. This means it's impractical to use the test filter in combination with a multi-test gradle task.